### ISSUE TITLE
I have to call "bower install" several times

### ISSUE THREAD

**Issue by zxdev777** [bower/bower#1753]
I have to call bower install several times and after that target directory is populated with all libraries.

I tried to increase timeout in .bowerrc and also modify registry search (http:// not https://)

After every call "bower install" only few new libraries are copied to: bower\cache\packages
So after 3x times call bower install libraries appears in cache and target directory (directory in .bowerrc).

I am behind a corporate proxy

Does anyone experience the same issue ?

My env:

OS: MS Windows 7 Enterprise (V6.1.7601 SP1 7601)
bower 1.3.12
npm 1.4.28
node v0.10.35

**Comment by sheerun** [comment]
Any output, any errors? We need a way to reproduce to fix this.

**Comment by fazy** [comment]
I have a similar issue that started happening at around the same time as the DDOS attack against Github that is still ongoing.

bower.json:

```
{
  "name": "pwncg-jquery-lib",
  "dependencies": {
    "foundation": "zurb/bower-foundation#~5.5",
    "jquery-legacy": "jquery#~1.11",
    "jquery-modern": "jquery#~2.1",
    "html5shiv": "html5shiv#~3.7",
    "nwmatcher": "dperini/nwmatcher#1.3",
    "selectivizr": "keithclark/selectivizr#ed2f5e3e56",
    "respond": "scottjehl/Respond#1.4.2",
    "es5shim": "es-shims/es5-shim#4.0.5",
    "remjs": "chuckcarpenter/REM-unit-polyfill#1.3.2",
    "slick.js": "~1.4.1",
    "datetimepicker": "~2.4.1",
    "parsleyjs": "parsleyjs#~2.0",
    "velocity": "~1.2.2"
  }
}
```

Logs here: http://pastebin.com/nQnPWhqy

As far as I can see Bower is not retrying after getting connection refused. Are there any options I can use to improve the retries on individual packages, so that it can get through a full run?

I don't mind just running the Bower command in my build scripts a few times, but after manually retrying even 5 times it doesn't get through to the end.

**Comment by zxdev777** [comment]
I tried it today and it works. No modification on my side.. :-/

Unfortunately I didn’t run it with verbose switch before, but I didn’t see any 403 status code in my console.

It would be nice if Nexus will have also option for bower to proxy it as remote repository.
We upgraded our nexus and support for NPM is already there.

So for now we are realizing to use private bower as good option for cache public bower registry and temporary outage.
